uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -674,6 +678,63 @@ public void clearStorage() {
         log.info("Storage was cleared");
     }
 
+    /**
+     * Sets MQTT user Properties to transmit.
+     *
+     * @param key the key of userProperties property.
+     * @param value the value of userProperties property.
+     */
+    @And("I set MQTT user property with key {string} and value {string} to transmit")
```

Comment:
Take user property in single quotes.

Better 
"I add MQTT 'user property' with key and value to transmit"

Rename method to addTxUserProperty